main/models.py

```diff
@@ -0,0 +1,71 @@
+from django.db import models
+from django.contrib.auth.models import User
+
+
+class UserProfile(models.Model):
```

Comment:
Why not inherit it from django's `AbstractUser`?

Reply:
https://docs.djangoproject.com/en/4.0/topics/auth/customizing/#extending-django-s-default-user

Reply:
I thought, OneToOneField is possible, fixed